### PR TITLE
Replace prompt with inquirer to have a clearer trust-asking prompt

### PR DESCRIPTION
**Pull request by ilesinge**
This is a solution for #5 

Benefits:

- Replace true/false answers with (more common) yes/no answers
- The question stand out in the terminal if colors/boldness are supported
- No "Invalid input for undefined" error

Less good:

- Inquirer does not allow certain special characters (such as ".") as part of a question key.

For this last point, I had to base64 encode the path to create the question key (and reverse it when getting the question results).

### CODE REVIEW

File: src/commands/verify.ts [hunk 1]
```diff
@@ -144,35 +134,26 @@ export default class extends Command {
                 }
                 if (prompts.filter((value) => basename(value.name) == result.packageName).length == 0) {
                     prompts.push({
-                        name: path,
-                        type: 'boolean',
-                        description: 'Package \'' + result.packageName + '\' is not trusted, but is signed by ' + identityString + '. ' + 
+                        name: Buffer.from(path).toString('base64'),
```
Comment (ilesinge) [inline src/commands/verify.ts:137]: meh...

Comment (hach-que) [reply src/commands/verify.ts:137]: not great, but it will have to do

**Comment by hach-que** [comment]
Thanks for this, this looks to be a much better solution than prompt (and yeah, the true/false options do suck, it just happened to be something quick I could throw in to get 0.1.0 working).

I'll look at merging this tomorrow (AEST).

**Comment by hach-que** [comment]
Thanks for this, I have merged it into dev.

**Pull request closed without merging**